tools/go-cli/go-whisk-cli/commands/package.go

```diff
@@ -185,16 +184,6 @@ var packageCreateCmd = &cobra.Command{
         }
         client.Namespace = qName.namespace
 
-        if (flags.common.shared == "yes") {
```

Comment:
The package we are using to parse arguments does not allow flags to to arguments, and not take arguments. When --shared is not present in a command, the entity will be set to not be shared.